sync-core/src/test/java/com/cloudant/sync/replication/BasicPushStrategyTest2.java

```diff
@@ -39,6 +39,9 @@
     String id2;
     String id3;
 
+    BasicPushStrategy push;
+    BasicPullStrategy pull;
```

Comment:
For me, `push()` and `pull()` should return the `Strategy` objects, rather than bunging them in class level variables, as they are created within a test method (as opposed to `setUp`).